envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -48,9 +48,13 @@ abstract class EnvoyControlTestConfiguration : BaseEnvoyTest() {
         var envoyControls: Int = 1
         var envoys: Int = 1
 
+        // We use envoy version from master. This is 1.14.0-dev.
```

Comment:
Could you please add TODO with issue numer as the other places so we can easily find those custom images later?

Reply:
But this is not related to that issue. This is a version of envoy we currently use (I just moved this comment, not added it)